### DISCUSSION TITLE
Integrate with Github Tags

### DISCUSSION THREAD

**Discussion by ImranBepari97** [GitbookIO/community#209] [Feature Requests]
### What's your feature/idea?

Currently, there's a feature to integrate with a live GitHub branch using `.gitbook.yml` to define the content and structure of the documentation while keeping it live. 

This is great, but we'd like to make sure older versions of our software are archived for later, using the same mechanism.

Since GitHub tags are effectively branches, it would be great to create pages in the same way as branched, but just import the content from a specific release. Even automating this would be amazing, as we could release our software against a tag and automatically have the documentation get archived alongside the latest docs.

### What's your desired outcome?

Use the Github integration to create documentation out of tags as well as branches for archiving and historical purposes.

The UI would ultimately look the same, you'd just get to pick from tags as well as branches in the drop down, and perhaps it would be entirely read only instead, as a tag cannot be edited. 

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

### Anything else?

This is important to our workflow, as our clients can have multiple different versions of our software deployed at once, even in the same teams.

**Comment by addisonschultz** [comment]
Hey @ImranBepari97, thanks for the feature request! I'd like to understand a bit more the use-case you're talking about. 

You're talking about [Git Sync](https://docs.gitbook.com/product-tour/git-sync), right? Definitely an interesting workflow—So you'd like to have different versions of your docs in GitBook for each one of your releases (tags), right?

**Reply by ImranBepari97** [reply]
Yes, via Git Sync! Yes, we'd like different versions of our docs in Gitbook for each of our Github releases, so it'd be great if Git Sync supported tags.
